Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,56 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponent_GravitySetsWhileMoving)
+    {
+        PhysX::CharacterControllerComponent* characterComponent = nullptr;
+        PhysX::CharacterGameplayComponent* gameplayComponent = nullptr;
+        AzFramework::TransformComponent* transform = nullptr;
+
+        float expectedGravityMultiplier = 2.5f;
+
+        // Create character
+        auto gameplayEntity = AZStd::make_unique<AZ::Entity>("GameplayEntity");
+        {
+            auto characterConfiguration = AZStd::make_unique<Physics::CharacterConfiguration>();
+            auto characterShapeConfiguration = AZStd::make_unique<Physics::CapsuleShapeConfiguration>();
+
+            characterShapeConfiguration->m_height = 1.5f;
+            characterShapeConfiguration->m_radius = 0.5f;
+
+            PhysX::CharacterGameplayConfiguration characterGameplayConfiguration;
+
+            characterGameplayConfiguration.m_gravityMultiplier = expectedGravityMultiplier;
+            characterGameplayConfiguration.m_groundDetectionBoxHeight = 0.05f;
+
+            transform = gameplayEntity->CreateComponent<AzFramework::TransformComponent>();
+            characterComponent = gameplayEntity->CreateComponent<CharacterControllerComponent>(
+                AZStd::move(characterConfiguration), AZStd::move(characterShapeConfiguration));
+            gameplayComponent = gameplayEntity->CreateComponent<CharacterGameplayComponent>(characterGameplayConfiguration);
+
+            transform->SetWorldTM(AZ::Transform::Identity());
+        }
+        gameplayEntity->Init();
+        gameplayEntity->Activate();
+
+        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
+        auto startTransform = transform->GetWorldTM();
```

Comment:
question: It doesn't look like this variable is being used anywhere, can it be removed?

Reply:
Was used in a previous version of this test and missed the cleanup when usage was removed. Thanks for catching this.

Removed.